index.html

```diff
@@ -50,5 +51,171 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
+          <br />
+          <button type="button" class="work-btn">See Project</button>
+        </div>
+      </div>
+      <div class="work-card">
+        <img
+          class="work-img"
+          src="./img/work/work-preview4.svg"
+          alt="work-preview"
+        />
+        <h3 class="work-name">Tonic</h3>
+        <div class="work-info">
+          <p class="info-main">CANOPY</p>
+          <i class="info-description info-sep fa fa-circle"></i>
+          <p class="info-description">Back End Dev</p>
+          <i class="info-description info-sep fa fa-circle"></i>
+          <p class="info-description">2015</p>
+        </div>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div>
+          <span class="lang-section">html</span>
+          <span class="lang-section">css</span>
+          <span class="lang-section">javascript</span>
+          <br />
+          <button type="button" class="work-btn">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!-- About MySelf section -->
+
+    <section class="about-me-section">
+      <div class="about-me-content">
+        <h2 class="about-me-header">About Myself</h2>
+        <p class="about-me-description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+          <br />
+          <a class="connect-link" href="#">Let's connect</a>
+        </p>
+
+        <ul class="social-links">
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-github-alt"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-linkedin-in"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-angellist"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-twitter"></i></a>
+          </li>
+
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-medium-m"></i></a>
+          </li>
+        </ul>
+        <button type="button" class="work-btn">Get my resume</button>
+
+        <div class="more-info">
+          <div class="collapsible-menu">
+            <button class="more-btn" type="button">Langguages</button>
+          </div>
+          <br />
+          <hr />
+          <br />
+          <div class="collapsible-menu">
+            <button class="more-btn" type="button">Frameworks</button>
+          </div>
+          <br />
+          <hr />
+          <br />
+          <div class="collapsible-menu">
+            <button class="more-btn" type="button">Skills</button>
+          </div>
+          <br />
+          <hr />
+        </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are a list and should use  a `ul`, each of the category should be in a `li`
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. You are currently missing this section of the page, kindly refer to the template and then add it.
![image](https://user-images.githubusercontent.com/49621279/120658081-f951ce00-c439-11eb-8362-3ab0b3b8c9e4.png)